assgnment2/funcions.dart

```diff
@@ -0,0 +1,183 @@
+
+import 'dart:convert';
+import 'util.dart';
+import 'user.dart';         /// two classes approach (util input, util validation)
+import 'dart:io';
+bool validateUserChoice(int userChoice){
```

Comment:
Don't add functions directly into a file. You can create a class and static methods to it